sparkmagic/sparkmagic/tests/test_command.py

```diff
@@ -125,10 +126,12 @@ def test_execute_failure_get_statement_output_emits_event():
         "status_sleep_seconds": 0.01,
         "statement_sleep_seconds": 0.01
     })
+    http_client.get_statement.return_value = tls.TestLivySession.ready_statement_json
```

Comment:
Can you please rename all the `_assert_called_once_with` to `assert_called_once_with`? Otherwise, tests are not actually verifying anything.